tests/periph_i2c/tests/periph_i2c.keywords.txt

```diff
@@ -0,0 +1,18 @@
+*** Settings ***
+Library             I2Cdevice  port=%{PORT}  baudrate=%{BAUD}  timeout=${10}
+
+Resource            riot_base.keywords.txt
+Resource            riot_util.keywords.txt
+Resource            api_shell.keywords.txt
+Resource            philip.keywords.txt
+
+*** Keywords ***
+DUT Must Have Periph I2C Application
+    API Call Should Succeed     I2C Get ID
```

Comment:
To prevent initial error from connecting and resetting I think it would be good to send the I2C Get ID command once without any checks first, then send and check the id.  Either include it here or do it before calling this keyword.

Reply:
okay, will add a simple `I2C Get ID` without checking the result before this line